### PR TITLE
Add link to GitOps Toolkit

### PR DESCRIPTION
**Pull request by dholbach**
closes: #17

### CODE REVIEW

File: layouts/partials/notification.html [hunk 1]
```diff
@@ -2,3 +2,4 @@
 <div class="notification is-warning mb-0">
   {{ $notification }}
 </div>
+
```
Comment (hiddeco) [inline layouts/partials/notification.html:5]: Obsolete change?

Comment (dholbach) [reply layouts/partials/notification.html:5]: I might not have `git add`ed it the last time... :facepalm: 

https://github.com/fluxcd/fluxcd.io/pull/25/files

Comment (hiddeco) [reply layouts/partials/notification.html]: Somehow resolved magically in the previous PR then.

**Review by hiddeco — APPROVED** [review]
Thank you @dholbach :star:

**Pull request merged**